csrc/apis/c/CMakeLists.txt

```diff
@@ -24,6 +24,9 @@ else ()
         list(APPEND TASK_LIST "text_detector")
         list(APPEND TASK_LIST "text_recognizer")
     endif ()
+    if ("mmpose" IN_LIST MMDEPLOY_CODEBASES)
+        list(APPEND TASK_LIST "pose_estimator")
```

Comment:
Inconsistent name: `pose_detector` vs `pose_estimator`

Reply:
changed to pose_detector